examples/discovery-search-app/src/App.js

```diff
@@ -3,8 +3,9 @@ import DiscoveryV2 from 'ibm-watson/discovery/v2';
 import { NoAuthAuthenticator } from 'ibm-watson/auth';
 import './app.scss';
 import { settings } from 'carbon-components';
-import { Button } from 'carbon-components-react';
+import { Button, Tabs, Tab } from 'carbon-components-react';
 import Close from '@carbon/icons-react/lib/close/16';
+import cx from 'classnames';
```

Comment:
make sure to add `classnames` as a dependency for the example app

Reply:
Good catch